mozilla_voice_tts/tts/utils/generic_utils.py

```diff
@@ -239,15 +248,33 @@ def check_config(c):
     # paths
     check_argument('output_path', c, restricted=True, val_type=str)
 
-    # multi-speaker gst
+    # multi-speaker and gst
     check_argument('use_speaker_embedding', c, restricted=True, val_type=bool)
-    check_argument('style_wav_for_test', c, restricted=True, val_type=str)
+    check_argument('use_external_speaker_embedding_file', c, restricted=True, val_type=bool)
+    check_argument('external_speaker_embedding_file', c, restricted=True, val_type=str)
     check_argument('use_gst', c, restricted=True, val_type=bool)
+    check_argument('gst_style_input', c, restricted=True, val_type=str)
```

Comment:
change this to:  
    check_argument('gst_style_input', c['gst'], restricted=True, val_type=str)

Reply:
I updated that part. There were multiple checks for the same arguments. Can you check again?

Reply:
Now everything looks fine :)